packages/eslint-plugin/docs/rules/prefer-ts-expect-error.md

```diff
@@ -32,6 +42,16 @@ Examples of **correct** code for this rule:
 // @ts-expect-error
 const str: string = 1;
 
+{
+  /* @ts-expect-error */
+}
+const value: number = 'value';
+
+/**
+ * @ts-expect-error
+ */
+const block: string = 1;
```

Comment:
this is not valid code - the expect error does not apply to the next line in either of these cases
[repl](https://www.typescriptlang.org/play?#code/N4KABGD0BUYAIBcDOBaApgDwA5oMYPQCdCB7QsaSEAXxFxIDskEwA3AQwBsBXNALjANuAWwBGacgF4wAcg480MgNwgY0cLESpMOfEVKENVekxajOJXAGsBzQgEsGAczDSAjEqA)

Reply:
Thanks for pointing it out.

I think I got mixed up with the following code [repl](https://www.typescriptlang.org/play?#code/PQKhAIChwqfAAgFwM4FoCWBzAdgewCcBTGYSANwEMDwqAbAVyIC5wUkCMctwBecAIxA)

```typescript
/** 
 *  
 * @ts-ignore */
let value: string = 1
```

Corrected and added additional test cases for the same.